app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    return_list = []
+    for person in people:
+        # creating extra variable to make the code easier
+        human = Person(person["name"], person["age"])
+        return_list.append(human)
+
+    for person in people:
+        if "wife" in person.keys() and person["wife"] is not None:
```

Comment:
1) person.keys() == person
2) better to use `get`
```suggestion
        if "wife" in person and person.get("wife") is not None:
```